tsconfig.esm.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "module": "ES2022"
+    "module": "ES2022",
```

Comment:
doesn't this break anything? it's not valid json

Reply:
https://json.schemastore.org/tsconfig

![image](https://github.com/wormhole-foundation/wormhole-sdk-ts/assets/520236/eef49bb3-1cb7-4995-814f-9e917c40a3d4)